From be82b68c8c9d4425f68902dc1de245de594632f6 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 1 Mar 2021 18:12:12 +0700
Subject: [PATCH 096/135] arm64: dts: renesas: r9a07g044l: add MIPI DSI device
 node

This commit adds support MIPI DSI Encoder device node for r9a07g044l.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l.dtsi | 30 +++++++++++++++++++++++++++++
 1 file changed, 30 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
index 525c3fb..0586015 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
@@ -466,6 +466,36 @@
 				port@1 {
 					reg = <1>;
 					du_out_dsi0: endpoint {
+						remote-endpoint = <&dsi0_in>;
+					};
+				};
+			};
+		};
+
+		dsi0: dsi@10860000 {
+			compatible = "renesas,r9a07g044l-mipi-dsi";
+			reg =	<0 0x10860000 0 0x10000>, /* LINK */
+				<0 0x10850000 0 0x10000>; /* DPHY */
+
+			clocks = <&cpg CPG_MOD R9A07G044L_CLK_MIPI_DSI>;
+			resets = <&cpg R9A07G044L_CLK_MIPI_DSI>;
+			power-domains = <&cpg>;
+			status = "disabled";
+
+			ports {
+				#address-cells = <1>;
+				#size-cells = <0>;
+
+				port@0 {
+					reg = <0>;
+					dsi0_in: endpoint {
+					remote-endpoint = <&du_out_dsi0>;
+					};
+				};
+
+				port@1 {
+					reg = <1>;
+					dsi0_out: endpoint {
 					};
 				};
 			};
-- 
2.7.4

